src/town_cmd.cpp

```diff
@@ -1978,7 +1978,14 @@ void UpdateTownRadius(Town *t)
 		{121, 81,  0, 49, 36}, // 88
 	};
 
-	if (_settings_game.economy.town_zone_calc_mode) {
+	if (_settings_game.economy.town_zone_calc_mode, t->larger_town) {
```

Comment:
Presumably this should be the && operator?
As written this uses the custom values for cities even when the setting is off.

Reply:
That's a brain fart and a half, will fix.

Reply:
```suggestion
	if (_settings_game.economy.town_zone_calc_mode && t->larger_town) {
```